### ISSUE TITLE
public static function allows multiple spaces around "static" keyword

### ISSUE THREAD

**Issue by amaisano** [squizlabs/PHP_CodeSniffer#3034]
**Describe the bug**
I was shocked to find no warnings were raised when my method declaration had tons of extra space before and after the "static" keyword. Same for space between "public" and "function."

**Code sample**
```php
...
public         static     function() ...
...
public                            function() ...
```

**Custom ruleset**
Using the Drupal coding standard standard.

**To reproduce**
1. Create a file called `test.php` with the code sample above...
2. Run `phpcs test.php ...`
3. Observe no errors/warnings have been raised!

**Expected behavior**
- Only 1 space should be allowed before and after the "static" keyword
- Only 1 space should be allowed between the "public" or "private" keywords and the "function" keyword

**Versions (please complete the following information):**
 - OS: MacOS 10.15
 - PHP: 7.3
 - PHPCS: 3.5.5, master
 - Standard: [drupal]

**Additional context**
Using VS Code with Intelephense Premium, phpcs, and phpcbf add-ons (the latter two use phpcs installed globally with composer).

**Comment by amaisano** [comment]
I double checked with the PSR2 standard, and I got the correct warnings. This is a problem with drupal/coder's XML rules, I will file a bug there.
